### ISSUE TITLE
Buildkite agent docker image for alpine linux broken

### ISSUE THREAD

**Issue by rjhazelwood** [buildkite/feedback#364]
The latest version buildkite/agent:3 (3.1.1) doesn't start up because tini is missing.

**Comment by toolmantim** [comment]
Sorry @rjhazelwood — we're getting this fixed now.

**Comment by toolmantim** [comment]
Ok, fixed images are up and that should be working as expected again.

```
$ docker pull buildkite/agent
Using default tag: latest
latest: Pulling from buildkite/agent
Digest: sha256:1e9fefd4b9575c8d12d4e994583a84e240790160a36f84cfad57f563845ce44b
Status: Image is up to date for buildkite/agent:latest
$ docker run --rm buildkite/agent --version
buildkite-agent version 3.1.1, build 2376
```

It appears [tini was removed](https://github.com/buildkite/agent/pull/749/files#diff-97b64c90fd588a75fc59f5d66e4d61d8L4) (in buildkite/agent#749) and [the tests](https://github.com/buildkite/agent/pull/749/files#diff-45a620c3d9025190c1ad98aa195ed646R26) override the default custom entrypoint, resulting in the tests incorrectly passing and sending it live without a working entrypoint.

We've fixed pushed new images, fixed master, and will investigate fixing the tests too.

Sorry again for this. And let me know if it's all working good for you now.
